modules/stochastic_tools/doc/content/source/distributions/StudentT.md

```diff
@@ -0,0 +1,21 @@
+# Student t-distribution
+
+!syntax description /Distributions/StudentT
+
+## Overview
+
+This object defines a [student t-distribution](https://en.wikipedia.org/wiki/Student%27s_t-distribution) with one degree of freedom parameter [!param](/Distributions/StudentT/dof) ($\nu$). The probability density function and cumulative distribution function are defined in [eq:t_pdf] and [eq:t_cdf], respectively.
+
+!equation id=eq:t_pdf
+f(x; \nu) = \frac{1}{\sqrt{\nu}\Beta\left(\frac{1}{2},\frac{\nu}{2}\right)} \left(1+\frac{x^2}{\nu}\right)^{-\frac{\nu+1}{2}},
```

Comment:
In the denominator, should it be `\sqrt{\pi~\nu}` and instead of `\Beta\left(...` should it be `\Gamma\left(`? I referred to https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.t.html for the definition (which seems to match with Wiki).

Reply:
Oh I see you have used the alternative definition, right?

Reply:
Yes, I thought this representation was a bit simpler. What do you think?

Reply:
Looks good to me